tests/mmm/test_delayed_saturated_mmm.py

```diff
@@ -872,27 +872,39 @@ def test_plot_new_spend_contributions_prior_select_channels(
 @pytest.fixture(scope="module")
 def fixed_model_parameters() -> dict[str, Union[float, list[float]]]:
     return {
-        "intercept": 2.5,
-        "beta_channel": [0.5, 0.5],
+        "intercept": 5.0,
+        "beta_channel": [0.15, 0.5],
         "alpha": [0.5, 0.5],
         "lam": [0.5, 0.5],
         "likelihood_sigma": 0.25,
-        "gamma_control": [0.5, 0.5],
+        "gamma_control": [0.0001, 0.005],
```

Comment:
Reduced these based on the max values of toy_X and trying to get reasonable model_generated_y